### PR TITLE
LPS-91866 PersistenceNestedSetsTreeManagerTest

### PR DESCRIPTION
**Pull request by dantewang**
@tinatian Moved to portal-tools-service-builder-test-test as Shuyang instructed. I removed CodeCoverageAssertor since Whip agent is not available when running Arquillian tests.

### CODE REVIEW

**Comment by dantewang** [comment]
ci:test:relevant

**Comment by dantewang** [comment]
ci:test:sf

**Comment by tinatian** [comment]
@dantewang 
Since we have portal-tools-service-builder-test-api and portal-tools-service-builder-test-service, please create your own api and related service to avoid using AssetCategory related stuff, and then move out the test.

Thanks
Tina

**Comment by tinatian** [comment]
@dantewang 
My bad, misunderstood your code.

Tina

File: modules/util/portal-tools-service-builder-test-test/src/testIntegration/java/com/liferay/portal/tools/service/builder/test/service/persistence/impl/test/PersistenceNestedSetsTreeManagerTest.java [hunk 1]
```diff
@@ -61,10 +62,13 @@
 public class PersistenceNestedSetsTreeManagerTest {
 
 	@ClassRule
+	@Rule
 	public static final AggregateTestRule aggregateTestRule =
 		new AggregateTestRule(
-			CodeCoverageAssertor.INSTANCE, new LiferayIntegrationTestRule(),
-			TransactionalTestRule.INSTANCE);
+			new LiferayIntegrationTestRule(),
```
Comment (tinatian) [inline modules/util/portal-tools-service-builder-test-test/src/testIntegration/java/com/liferay/portal/tools/service/builder/test/service/persistence/impl/test/PersistenceNestedSetsTreeManagerTest.java:68]: @dantewang You are removing the CodeCoverageAssertor one, this is changing the logic. Is it possible that we add CodeCoverageAssertor support for Arquillian test? There must be other integration tests using CodeCoverageAssertor

**Comment by tinatian** [comment]
See https://github.com/shuyangzhou/liferay-portal/pull/7504

**Pull request closed without merging**